include/sacn/merge.h

```diff
@@ -0,0 +1,162 @@
+/******************************************************************************
+ * Copyright 2020 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************
+ * This file is a part of sACN. For more information, go to:
+ * https://github.com/ETCLabs/sACN
+ *****************************************************************************/
+
+/*!
+ * \file sacn/merge.h
+ * \brief DMX Merge API definitions
+ *
+ * Functions and definitions for the \ref dmx_merger "DMX Merger API" are contained in this
+ * header.
+ */
+
+#ifndef DMX_MERGER_H_
+#define DMX_MERGER_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "etcpal/error.h"
+#include "etcpal/uuid.h"
+#include "sacn/common.h"
+#include "receiver.h"
+
+/*!
+ * \defgroup dmx_merger DMX Merger
+ * \ingroup sACN
+ * \brief The DMX Merger API
+ *
+ * This API provides a software merger for buffers containing DMX512-A start code 0 packets.
+ * It also uses buffers containing DMX512-A start code 0xdd packets to support per-address priority.
+ *
+ * When asked to calculate the merge of a universe, the merger shall evaluate the current source
+ * buffers and update two result buffers:
+ *   512 bytes for the merged data values (i.e. "winning level").  These are calculated by using
```

Comment:
If you prefix these lines with another * (or a -, or a +, or I think a few other "bullet" types are valid), Doxygen will convert them to bullets using markdown conventions.

FYI in case you weren't aware, the CI is set up to generate staging docs for each pull request. So you can check what the generated documentation for this PR looks like at the URL matching the number of this pull request:
https://etclabs.github.io/sACN/docs/stage/5

This paragraph is rendered at https://etclabs.github.io/sACN/docs/stage/5/group__dmx__merger.html#details

Reply:
Thanks for the heads up on the docs!